src/http_seed_connection.cpp

```diff
@@ -315,7 +315,7 @@ namespace libtorrent
 					}
 
 					// add the redirected url and remove the current one
-					t->add_web_seed(location, web_seed_entry::http_seed);
+					t->add_web_seed(location, web_seed_entry::http_seed, std::string(), web_seed_entry::headers_t(), true);
```

Comment:
redirected web_seed should not be stored in resume because it is only serialize url part (without "redirects" map") e.g original url points to "http://example1.com/file1", web server response with location "http://example2.com/subpath/file2". (m_url=="http://example2.com/", redirects[0]=="/subpath/file2"). if we try to load old save resume then library tryies to request wrong path "http://example2.com/file1" ("redirects" map is lost).  alternative approach is to serialize "redirects" maps, but it will definetly increase resume data size.